gq/gq_test.go

```diff
@@ -235,6 +236,10 @@ func getClaimInProtected(claimKey string, token []byte) (any, bool, error) {
 		return nil, false, err
 	}
 
-	claimValue, ok := headers.Get(claimKey)
-	return claimValue, ok, nil
+	var claimValue string
+	err = headers.Get(claimKey, &claimValue)
+	if err != nil {
+		return nil, false, nil
```

Comment:
Thinking about returning the error as below, but your current version preserves the original behavior so it is better than my suggestion. Ignore my suggestion and add a comment here since everyone expects an error to be returned when an error happens. Good catch!

```suggestion
		return nil, false, err
```